.github/workflows/cmake.yml

```diff
@@ -245,6 +245,31 @@ jobs:
       working-directory: ${{github.workspace}}/build
       run: ctest -C ${{matrix.build_type}} --output-on-failure -L "conformance" --timeout 180
 
+  ubuntu-adapter-test:
+    name: Conformance test - Adapter L0 on devcloud Ubuntu
+    strategy:
+      fail-fast: false
+      matrix:
+        adapter: [L0]
```

Comment:
you don't need this param as part of the `matrix` - perhaps use an env, or just skip this param at all - for now we'll only run L0, anyway